pkgs/applications/audio/sony-headphones-client/default.nix

```diff
@@ -19,10 +19,9 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-Wno-dev" ];
 
-  postPatch = ''
-    substituteInPlace Constants.h \
-      --replace "UNKNOWN = -1" "// UNKNOWN removed since it doesn't fit in char"
-  '';
+  patches = [
+    ./fix-aarch64.patch
```

Comment:
Huh, can't we `fetchpatch` the patch?

Reply:
Can you try to change into the sourceRoot in postPatch?

Reply:
that is easier said than done. I do not know how to easily fix this.

Reply:
please consider this provided change as a fix. All other solutions will have much more changes.

Reply:
So I guess we will wait for the next release. Fixing the build seems not important...